### PR TITLE
Update chapter.md

### PR DESCRIPTION
**Pull request by fakela**

### CODE REVIEW

**Comment by fakela** [comment]
@RCheesley  please check on this. Thanks

File: pages/09.education-team/chapter.md [hunk 1]
```diff
@@ -10,6 +10,15 @@ The Education Team is responsible for the Forum, Mautic Documentation and Intern
 
 > To improve the resources available for new and established users to learn how to use Mautic
 
+### General Tasks
+
+  * Help marketers to contribute their existing content to the Mautic knowledge base
```
Comment (RCheesley) [inline pages/09.education-team/chapter.md:15]:
```suggestion
  * Help marketers to contribute their existing content to the [Mautic Knowledgebase](https://kb.mautic.org)
```

So far we have used all one word and capitalised - not sure if that is correct, maybe we should use two separate words! :o

File: pages/09.education-team/chapter.md [hunk 2]
```diff
@@ -10,6 +10,15 @@ The Education Team is responsible for the Forum, Mautic Documentation and Intern
...
+  * Update the documentation to be up to date with new releases
```
Comment (RCheesley) [inline pages/09.education-team/chapter.md:16]:
```suggestion
  * Ensure that the documentation is up to date when new releases are made
```

File: pages/09.education-team/chapter.md [hunk 3]
```diff
@@ -10,6 +10,15 @@ The Education Team is responsible for the Forum, Mautic Documentation and Intern
...
+  * Improve users experience by making the documentation high quality and comprehensive
```
Comment (RCheesley) [inline pages/09.education-team/chapter.md:17]:
```suggestion
  * Improve the experience of Mautic users by ensuring that the documentation is of high quality and comprehensive
```

File: pages/09.education-team/chapter.md [hunk 4]
```diff
@@ -10,6 +10,15 @@ The Education Team is responsible for the Forum, Mautic Documentation and Intern
...
+  * Moderate forums and provide contents to FAQs
```
Comment (RCheesley) [inline pages/09.education-team/chapter.md:18]:
```suggestion
  * Moderate the [Mautic Community Forums](https://forum.mautic.org) and provide content to answer FAQs
```

**Review by RCheesley — CHANGES_REQUESTED** [review]
Just a couple of suggestions to improve the flow :) Thanks so much for making the update!

**Pull request merged**